server/controllers/course-controller.js

```diff
@@ -0,0 +1,37 @@
+const knex = require("./../db");
+
+// Get the list of lectures scheduled for a course
+exports.getScheduledLectures = async (req, res) => {
```

Comment:
I think that is a bit confusing that a controller about courses returns a list of lectures, shouldn't the lecture-controller handle this case?

Reply:
Resolved